examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -36,6 +37,14 @@
 from examples_utils.benchmarks.slurm_utils import (check_slurm_configured, configure_slurm_job,
                                                    run_and_monitor_progress_on_slurm)
 
+try:
+    from .monitoring_utils import plot_ipu_usage
+except (ImportError, ModuleNotFoundError) as error:
+
+    def plot_ipu_usage(*args, **kwargs):
```

Comment:
is this needed? the only calls I can see are under a condition anyway, so would it ever happen that the actual function cant be imported but monitor_ipus and gc_monitor are True?

Reply:
Ah yes, that situation happens: the try except is triggered whenever examples utils is installed without pandas (without the `[jupyter]` variant), but there is noting stopping a user from doing `--gc-monitor` without the jupyter variant. In that case, the log files are still generated but we cannot plot the data because we don't have the right dependencies.

I think I'll add a comment explaining that before merging.